### PR TITLE
Hook to save house data in Firebase

### LINKED ISSUE #44: Add House
## DESIGN

Captura de pantalla 2023-08-14 173418

### PR DESCRIPTION
**Pull request by FebernalGober**
The Hook useAddHouse was created to add the data of the houses to firebase

# Description

### CODE REVIEW

File: renta_facil_colima/src/styles/AddHouse.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import {StyleSheet} from 'react-native';
+export const AddHouseStyles = StyleSheet.create({
+
+
+
+
+
+});
```
Comment (BrandonVG) [inline renta_facil_colima/src/styles/AddHouse.js]: Delete this file if is not neccesary

File: renta_facil_colima/src/hooks/useAddHouse.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    address: address,
+    rooms: rooms,
+    bathrooms: bathrooms, 
+    surface: surface,
+    rent: rent,
+  });
+
+  console.log("Document written with ID: ", docRef.id);
+} catch (e) {
+  console.error("Error adding document: ", e);
```
Comment (BrandonVG) [inline renta_facil_colima/src/hooks/useAddHouse.js:28]: Return the data if success and use an Alert.alert on error

File: renta_facil_colima/src/hooks/useAddHouse.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import { getFirestore, collection, addDoc } from "firebase/firestore";
+import { firebaseConfig } from "../Firebase/FirebaseConfig";
+
+
+
+// Initialize Firebase
+const app = initializeApp(firebaseConfig);
+
+
+// Initialize Cloud Firestore and get a reference to the service
```
Comment (BrandonVG) [inline renta_facil_colima/src/hooks/useAddHouse.js:11]: Delete comments

**Review by BrandonVG — CHANGES_REQUESTED** [review]
image

**Review by BrandonVG — APPROVED** [review]

**Review by Lizzflores — APPROVED** [review]

**Review by Alfredo20164920 — APPROVED** [review]

**Pull request merged**